DEBIAN/changelog

```diff
@@ -0,0 +1,5 @@
+gcsfuse (1.0.0) stable; urgency=medium
+
+  * Package created with dpkg-deb --build
+
+ -- GCSFuse Eng Team <gcs-fuse-eng@google.com>  Thu, 13 Jul 2023 05:37:50 +0000
```

Comment:
is it okay to write google ids here?

Reply:
I believe it should be okay. We should add maintainer's name and email id here.
Ref: 
https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog
https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#changelog